lib/cc/analyzer/container.rb

```diff
@@ -67,6 +67,12 @@ def run(options = [])
         end
       end
 
+      def stop
+        @on_output = ->(*) { }
+        Process.kill("TERM", @pid)
```

Comment:
I think this should be resilient to being called prematurely. Would a simple `if @pid` achieve that?